deployment/network-operator/values.yaml

```diff
@@ -34,15 +38,16 @@ node-feature-discovery:
         operator: "Equal"
         value: "present"
         effect: "NoSchedule"
-    options:
+    config: |
       sources:
-        pci:
-          deviceClassWhitelist:
-            - "02"
-            - "0200"
-            - "0207"
-          deviceLabelFields:
-            - vendor
+          pci:
```

Comment:
An indentation looks to big but it works as expected

Reply:
@e0ne  please open an issue in github and in our RM for this. 
I will merged as it has it doesn't effect functionality